api/Navigator.json

```diff
@@ -3259,10 +3271,61 @@
             }
           },
           "status": {
-            "experimental": true,
+            "experimental": false,
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "files": {
```

Comment:
Can we give this a richer name and description? For the name, perhaps named `data_parameter_files` and something like `"Files as a <code>data</code> paramter"`? Riffing on [the parameter naming conventions](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#parameters-and-parameter-object-features), mainly.

Reply:
That sounds like `.share(files)` but this is more like `.share({ title, text, files })`;

Reply:
ohhh, I misread the docs. I saw "Possible values" and thought it was one argument.

The correction here would be to closely follow the data guideline: `data_files_parameter` with description  `"<code>data.filescode> parameter"`.